home.css

```diff
@@ -0,0 +1,731 @@
+/* @import "https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap";
+  @import "https://fonts.googleapis.com/css2?family=Lato:wght@100&display=swap"; */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.logo img {
+  width: 48px;
+  display: none;
+}
+
+.menu-toggle {
+  display: none;
+  cursor: pointer;
+}
+
+.menu-img-container {
+  float: left;
+  margin-right: 10px;
+}
+
+.menu-img {
+  width: 11%;
+  margin-left: 25px;
+  margin-top: -12px;
+  margin-bottom: 41px;
+}
```

Comment:
- [ ] `Requred`
- [ ] You are doing an amazing job so far! 👏 
- [ ] However, when i tried to click on the humburger it never opened as expected instead ,the humburger just disappeared
- [ ] please  make amends